app/cli/cmd/attestation_status.go

```diff
@@ -83,6 +84,9 @@ func attestationStatusTableOutput(status *action.AttestationStatusResult, full b
 	gt.AppendSeparator()
 	meta := status.WorkflowMeta
 	gt.AppendRow(table.Row{"Attestation ID", status.AttestationID})
+	if status.Digest != "" {
```

Comment:
I moved the digest inside since it was affecting with blocking in the case of json output.

This makes it also more consistent